scripts/zkai_attention_kv_stwo_binary_typed_proof_accounting_gate.py

```
@@ -0,0 +1,685 @@
+        "run",
+        "--locked",
+        "--features",
+        "stwo-backend",
+        "--bin",
+        "zkai_stwo_proof_binary_accounting",
+        "--",
+        "--evidence-dir",
+        str(EVIDENCE_DIR),
+        *[str(path) for path in expected_envelope_paths()],
+    ]
+    completed = subprocess.run(command, cwd=ROOT, check=False, text=True, capture_output=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)